backend/mlarchive/tests/archive/utils.py

```diff
@@ -375,3 +376,25 @@ def test_get_subscribers(mock_client):
         (response_b, response_b.json())]
     subs = get_subscribers('public')
     assert subs == ['holden.ford@example.com']
+
+
+def test_purge_incoming(tmpdir, settings):
+    print('tmpdir: {}'.format(tmpdir))
```

Comment:
Did you leave this `print()` in on purpose or is it leftover debug?

Reply:
Correct. I had a habit of doing this, but should delete once the test is working properly as it muddles the code.